src/index.js

```diff
@@ -613,18 +708,18 @@ export default class Matic {
       !options.nonce
         ? web3.eth.getTransactionCount(from, 'pending')
         : options.nonce,
-      !options.chainId ? web3.eth.net.getId() : options.chainId,
+        !options.chainId ? web3.eth.net.getId() : options.chainId,
     ])
-
     return {
       from,
-      gasLimit,
       gas: gasLimit,
-      gasPrice: gasPrice,
+      gasLimit,
```

Comment:
what's `gasLimit` and `gas`? @ptsayli

Reply:
basically both are same
earlier we're facing issue some web3 version accepting `gas` and other `gasLimit`
so here we are passing both parameters